scss/menu.scss

```diff
@@ -0,0 +1,332 @@
+.menu-nav {
+  position: fixed;
+  left: 5%;
+  top: var(--nav-height);
+  width: 90%;
+  box-shadow: 0 2px 6px rgba(65, 11, 16, 0.15);
+  border-radius: 10px;
+  overflow: hidden;
+  background-color: var(--container-color);
+  transform: translateY(-365px);
+  transition: 0.3s;
+}
+
+.menu-nav.active {
+  transform: translateY(10px);
+}
+
+ul {
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  align-items: center;
+  overflow: hidden;
+  background-color: var(--container-color);
+}
+
+.navigation-li {
+  margin: 15px 0;
+  transition: 0.2s;
+}
+
+.navigation-li:nth-child(1) {
+  border-bottom: 2px solid var(--first-color);
+}
+
+.navigation-li:hover {
+  transform: scale(1.1, 1.1);
+}
+
+.navigation-a {
+  text-decoration: none;
+  color: var(--title-color);
+}
+
+.navigation-a:hover {
+  color: var(--first-color);
+}
+
+.navigation-li:nth-of-type(1) {
+  margin-top: 30px;
+}
+
+.navigation-li:nth-child(4) {
+  margin-bottom: 30px;
+}
+
+.navigation {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: var(--nav-height);
+  background-color: var(--container-color);
+  box-shadow: 0 2px 2px -2px var(--first-color);
+  z-index: 3;
+}
+
+.navigation-text {
+  margin-left: 20px;
+  color: var(--title-color);
+}
+
+.bars {
+  margin-right: 20px;
+  cursor: pointer;
+}
+
+.change-theme {
+  position: relative;
+  width: 48px;
+  height: 24px;
+  border: 3px solid black;
+  border-radius: 15px;
+  margin-right: 100px;
+  background-color: rgb(235, 230, 230);
+  margin: 0 auto 25px auto;
+  cursor: pointer;
+
+  @media (min-width: 620px) {
```

Comment:
You already have it in MediaQueries file

Reply:
Removed